tests/components/vacuum/test_xiaomi_miio.py

```diff
@@ -16,14 +16,16 @@
     ATTR_DO_NOT_DISTURB_START, ATTR_DO_NOT_DISTURB_END, ATTR_ERROR,
     ATTR_MAIN_BRUSH_LEFT, ATTR_SIDE_BRUSH_LEFT, ATTR_FILTER_LEFT,
     ATTR_CLEANING_COUNT, ATTR_CLEANED_TOTAL_AREA, ATTR_CLEANING_TOTAL_TIME,
-    CONF_HOST, CONF_NAME, CONF_TOKEN, PLATFORM,
+    CONF_HOST, CONF_NAME, CONF_TOKEN,
     SERVICE_MOVE_REMOTE_CONTROL, SERVICE_MOVE_REMOTE_CONTROL_STEP,
     SERVICE_START_REMOTE_CONTROL, SERVICE_STOP_REMOTE_CONTROL)
 from homeassistant.const import (
     ATTR_ENTITY_ID, ATTR_SUPPORTED_FEATURES, CONF_PLATFORM, STATE_OFF,
     STATE_ON)
 from homeassistant.setup import async_setup_component
 
+PLATFORM = 'xiaomi_miio'
```

Comment:
Don't use this, just import DATA_KEY ?

Reply:
Are you sure? Is the component/platform name needed here or the DATA_KEY: https://github.com/syssi/home-assistant/blob/0222ed9eae83d72650762efc8d0e190a3d61590a/tests/components/vacuum/test_xiaomi_miio.py#L118

Reply:
oh ok. Sorry.